src/components/ExpandableSectionButton/ExpandableSectionButton.tsx

```diff
@@ -19,10 +20,12 @@ const Wrapper = styled.div`
 `
 
 const ExpandableSectionButton: React.FC<ExpandableSectionButtonProps> = ({ onClick, expanded }) => {
+  const TranslateString = useI18n()
+
   return (
     <Wrapper aria-label="Hide or show expandable content" role="button" onClick={() => onClick()}>
       <Text color="primary" bold>
-        {expanded ? 'Hide' : 'Details'}
+        {expanded ? 'Hide' : TranslateString(658, 'Details')}
```

Comment:
Please wrap `Hide` in the translate function as well. If we don't have an ID already just use 999

Reply:
Done